evennia/prototypes/spawner.py

```diff
@@ -716,7 +716,7 @@ def spawn(*prototypes, **kwargs):
         val = prot.pop("tags", [])
         tags = []
         for (tag, category, data) in val:
-            tags.append((init_spawn_value(val, str), category, data))
+            tags.append((init_spawn_value(tag, str), category, data))
```

Comment:
Hmmm ... 
[comment](https://github.com/evennia/evennia/pull/1653#issuecomment-427663883) 

Didn't you pick this change on your spawner fix @Griatch?

Reply:
I did, yes. Not sure why it's still there. Fixed (again) in master now :)